plumgrid_playbooks/roles/plumgrid_common/tasks/main.yml

```diff
@@ -45,51 +40,37 @@
     file: /tmp/GPG-KEY
     state: present
 
-# Update repositories
-- name: Running apt-update
```

Comment:
@arbabnazar sorry for getting to this so late. Thanks for your contribution.

You seem to have removed the apt-get update task here, which is needed to ensure latest packages. Rest looks good.